### ISSUE TITLE
Stat boosting Tactics wearing out

### ISSUE THREAD

**Issue by Mordefal** [WarEmu/WarBugs#14003]
**Expected behavior and actual behavior:**
- Stat boosting tactics (for example Discipline, bonus Willpower for Zealot) effect just wears out every once in a while, especially after Scenarios. My Choppa friend is struggling with the same issue with his Strength boosting, corresponding tactic.

**Steps to reproduce the problem:**
- When you return from a Scenario, very often your stat boosting tactics' don't affect on your total stat value on the character page. When you deactivate the tactic and put it back on, it works again.

**Screenshots/Videos or archive.org evidences:**
- I don't think a screenshot would add any value to this.

Best regards,
Mordefal, a Zealot of Destruction

**Comment by sioding** [comment]
And you checked the same with the .getstats commaned? Just to make sure we have no paperdoll problem?

**Comment by Tesq** [comment]
IT'S NOT related only to stats even other kind of tactics such warped flesh (just one exemple may stop to work. It can happen after every zone swap you need to often swap tactics in/out.

**Comment by sioding** [comment]
ok then i need screenshots

**Comment by Mordefal** [comment]
Yes, that's what I was afraid of but didn't want to mix it with this because I wasn't sure. It's just not as easy to notice other Tactics starting to disfunction. But thank you Tesq for supplementing the issue. And yes as I said I've been connecting the issue to the zone swaps as well. I will try to get some screenshots.

**Comment by Mordefal** [comment]
Okay so here's how it seems to work:
Whenever I come out of a Scenario and GO TO COMBAT, my Tactics wear off and on and off and on during the combat. And when I take off my Discipline Tactic and put it back on, it stops doing that until I come out of a Scenario again.

BTW, I can't seem to .getstats in Combat.

![ss2](https://user-images.githubusercontent.com/50245189/57164933-4b107000-6dfe-11e9-92e7-4a1103203def.png)
![ss1](https://user-images.githubusercontent.com/50245189/57164934-4b107000-6dfe-11e9-9b81-1dac8bc4f50a.png)

**Comment by Ootootoo** [comment]
Yeah, I got used to rebuff and switch tactics when changing zones, even in the same RvR lake. Lets say that you enter in The Brighted Isle in the RvR zone, then move south to enter Chrace, or in Troll Country then move to Ostland. Most of the times, in the second that you enter in Chrace or Ostland and the minimap switch, your buff bar and tactics are reseted.

I have only noticed in the stats tactics and buffs that imply stat changes, but is most likely that it happens with an array of them related to character numbers. It can certainly be annoying.

I have not tested if it has something to be with the bolster, cause the normal PvE zones have less impact on it, and it happens way less times in them. Still happen though, but may be cause the tactic/buff was applied when still bolstered. Never really checked though, cause it's so obvious that I thought that someone had reported it, and I'm surprised that nobody did. I mean, all your buffs just fade away instantly.

My reason to presume that has to be with the bolster is that potions, that have fixed increases to stats where your level is irrelevant, seem to work perfectly and they don't fade away neither their effect in character sheet disappear.

**Comment by sioding** [comment]
Now we need to figure out if its only in these zones when you are bolstered. Can someone test it with a level 40 char?

**Comment by Mordefal** [comment]
Unfortunately I do not have a 40 character, but I do experience the issue in many zones outside Bolstering too. But hopefully someone can do testing with a 40 lvl character.

Best Regards,
Mordefal, a Zealot of Destruction

**Comment by Ootootoo** [comment]
You should take the ultimatum out of the issue, The screenshots currently prove the bug.

**Comment by sioding** [comment]
Sorry nope - need more tests. Otherwise its not enough information to let a dev look into it,

**Comment by Ootootoo** [comment]
I would like to simply take a video showing it cause you will see all the buffs from the bar simply fading away, but my computer will crash trying. I'm running RoR at the lowest possible graph config, and even effects on myself make the game unplayable, lets not speak of having a demanding program active in the background. I can't even have the web explore active,

If anybody can do what I said, entering an RvR lake then buffing and switching zones while under bolster and having stat boosting tactics active, and take a video with the character sheet open, that would be really nice. I just can't :(

**Comment by sioding** [comment]
I only ask for another test on a level 40 char.

**Comment by Mordefal** [comment]
Still have no lvl 40 character but adding this:
- My Choppa friend's Strength Tactic toggless off when he enters the zone of a Chosen's Strength Aura. And this happened in the IC nowhere near combat or any bolstering. So this does seem like a multidimensional problem to me.

Best Regards,
Mordefal, a Zealot of Destruction

**Comment by sioding** [comment]
Does this still happening?

**Comment by Tesq** [comment]
I Didn't note anything strange on my chosen yest but i didn't test this troughtly enough. Especially on my magus were the issues was prevalent than my chosen.

**Comment by Mordefal** [comment]
> Does this still happening?

Yes, it's exactly the same. Still having to reslot my Tactic.

**Comment by Mordefal** [comment]
> Does this still happening?

Yes. Aaand here's another thing that I noticed now: my Discipline Tactic seems to boost my Strength instead of Willpower lol. Reslotting the tactic corrects it again.
![bug1](https://user-images.githubusercontent.com/50245189/57736976-a3732780-76b2-11e9-8537-53296c8b5a55.png)
![bug2](https://user-images.githubusercontent.com/50245189/57736982-a837db80-76b2-11e9-9b5a-fa0166b15218.png)
